sqlalchemy_file/file.py

```diff
@@ -118,6 +130,7 @@ def save_to_storage(self, upload_storage: Optional[str] = None) -> None:
     def store_content(
         self,
         content: Any,
+        content_path: Optional[str] = None,
```

Comment:
It might be better to move the `content_path` parameter down. This way, existing calls to the function won't break